app/services/broadcasts/welcome_notification/generator.rb

```diff
@@ -1,6 +1,8 @@
 module Broadcasts
   module WelcomeNotification
     class Generator
+      NOTIFICATIONS_SET_LIVE_DATE = "2020-04-05 00:00:00 UTC".to_date
```

Comment:
This is more of a dummy date

Reply:
Could we make this an ENV var? Would make it pretty straightforward for us to set the feature "live" this week, and it would also make it much easier to toggle on/off in the future, too.

Reply:
I moved it to SiteConfig. Let me know what you think